src/api/app/components/build_results_monitor_component.html.haml

```diff
@@ -35,7 +35,7 @@
                         project_package_repository_binaries_path(project_name: project_name,
                                                                  package_name: package_name,
                                                                  repository_name: repository_name),
-                        title: "Binaries for #{repository_name}", class: 'build-result-name text-break')
+                        title: "Binaries for #{repository_name}", class: 'build-result-name')
```

Comment:
We'll might meet a different corner case where the repo name is very long we might want to handle differently, but I think it is fine for now and we can wait until that happens, I don't have a real case in mind for now.